src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/recordtypeparametername/Example1.java

```diff
@@ -0,0 +1,20 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RecordTypeParameterName"/>
+  </module>
+</module>
+*/
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.naming.recordtypeparametername;
+
+// xdoc section -- start
+class Example1 {
+  record MyRecord1<T>() {}
+
+  record MyRecord2<t>() {} // violation,
+  // the record type parameter name should match the regular expression "^[A-Z]$"
```

Comment:
Please use `// violation below ` for all cases. You can always cut message at the beginning or end to satisfy line length

Reply:
Done